docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,784 @@
+
+[[selecting-inplace-strategy]]
+=== Selecting InPlace strategy
+
+To select the InPlace strategy, set the spec.updateStrategy.type value in the Istio resource to InPlace.
+
+**Example specification to select InPlace update strategy**
+
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  version: v1.26.3
+  updateStrategy:
+    type: InPlace
+----
+
+You can set this value while creating the resource or edit it later. If you edit the resource after creation, make the change before updating the Istio control plane.
+
+When using ambient mode with the InPlace strategy, ensure that all three components (Istio, IstioCNI, and ZTunnel) are updated to compatible versions. The update should be performed in the following order:
+
+1. Istio control plane
+2. IstioCNI
+3. ZTunnel
+
+Running the Istio resource in High Availability mode to minimize traffic disruptions requires additional property settings. For more information, see link:../general/istiod-ha.adoc#running-istiod-in-ha-mode[Istiod HA guide].
+
+[[updating-istio-control-plane-with-inplace-strategy]]
+=== Updating Istio control plane with InPlace strategy
+
+When updating Istio using the InPlace strategy in ambient mode, you can increment the version by only one minor release at a time. To update by more than one minor version, you must increment the version after each update. The update process is complete after all ambient components (Istio control plane, IstioCNI, and ZTunnel) have been updated.
+
+[[update-the-istio-control-plane]]
+==== Update the Istio Control Plane
+
+You can update the Istio control plane version by changing the version in the Istio resource. When using the InPlace update strategy, the Sail Operator replaces the existing istiod deployment with a new version. The old istiod pod is terminated and a new pod with the updated version is created. Ambient workloads automatically reconnect to the new control plane once it becomes ready.
+
+*Prerequisites:*
+
+* You have cluster-admin access to your Kubernetes cluster.
+* You have installed the Sail Operator.
+* You have deployed Istio in ambient mode with the InPlace update strategy.
+* The Istio resource named `default` is deployed in the `istio-system` namespace with the InPlace update strategy.
+* You have verified the current version and the Istio resource is in a Healthy state by running the following command:
+
+[source,bash]
+----
+$ kubectl get istio -n istio-system
+NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   1           1       1        default           Healthy   v1.26.3   7d
+----
+
+*Procedure:*
+
+. Update the Istio resource version. For example, to update to Istio 1.26.4, set the `spec.version` field to `v1.26.4` by running the following command:
++
+[source,bash]
+----
+$ kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.26.4"}}'
```

Comment:
As @fjglira mentioned please check if we can use `istio_latest_version` instead of hardcoded version.